From 2180fa989538154632755b707ac109ed78b66f6a Mon Sep 17 00:00:00 2001
From: Guy Nisan <gnisan@habana.ai>
Date: Tue, 25 Aug 2020 22:31:27 +0300
Subject: [PATCH] [SW-17562] efuse: set 64 words per bank

Small fixup for Gaudi eFuse read - 64 words per bank instead of 16

Change-Id: I11321e15c0d8e91a13d980696302180029f16f76
Signed-off-by: Guy Nisan <gnisan@habana.ai>
---

diff --git a/gaudi/common.h b/gaudi/common.h
index 5a7fee2..749f983 100644
--- a/gaudi/common.h
+++ b/gaudi/common.h
@@ -36,7 +36,7 @@
 #define mmNIC2_TPC_PLL_BASE		0xD7A000
 #define mmNIC3_TS_BASE			0xDB8000
 
-#define FUSE_WORDS_PER_BANK		0x10
+#define FUSE_WORDS_PER_BANK		64
 #define I2C_SLAVE_ADDRESS		0x4f
 
 #ifndef __ASSEMBLY__
